Source/WebCore/PAL/ThirdParty/libavif/tests/oss-fuzz/avif_decode_fuzzer.cc

```diff
@@ -4,6 +4,8 @@
 #include "avif/avif.h"
 #include "avif/avif_cxx.h"
 
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t * Data, size_t Size);
```

Comment:
Yes my submission [275291@main](https://commits.webkit.org/275291@main) removed this line. Sorry!

Reply:
It's okay.  That introduced a warning, but for various reasons, `-Wno-error` is enabled on fuzzer builds.

The issue was adding `repro_fuzz.cc` to the compiled sources for the `avif_decoder_fuzzer` target, which didn't fail at build time, but failed at runtime.